dart/lib/src/enricher/_io_enricher.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+import 'dart:io';
+
+import '../protocol.dart';
+
+import 'enricher.dart';
+
+final Enricher instance = IoEnricher();
+
+/// Enriches [SentryEvents] with various kinds of information.
+/// Uses Darts [Platform](https://api.dart.dev/stable/dart-io/Platform-class.html)
+/// class to read information.
+class IoEnricher implements Enricher {
+  @override
+  FutureOr<SentryEvent> apply(SentryEvent event, bool hasNativeIntegration) {
+    // If there's a native integration available, it probably has better
+    // information available than Flutter.
+    final os = hasNativeIntegration
+        ? null
+        : _getOperatingSystem(event.contexts.operatingSystem);
+    final device =
+        hasNativeIntegration ? null : _getDevice(event.contexts.device);
+
+    final contexts = event.contexts.copyWith(
+      operatingSystem: os,
+      device: device,
+      runtimes: _getRuntimes(event.contexts.runtimes),
+    );
+
+    contexts['dart_context'] = _getDartContext();
+
+    return event.copyWith(
+      contexts: contexts,
+    );
+  }
+
+  List<SentryRuntime> _getRuntimes(List<SentryRuntime>? runtimes) {
+    // Pure Dart doesn't have specific runtimes per build mode
+    // like Flutter: https://flutter.dev/docs/testing/build-modes
+    final dartRuntime = SentryRuntime(
+      name: 'Dart',
+      rawDescription: Platform.version,
```

Comment:
> An unprocessed description string obtained by the runtime. For some well-known runtimes, Sentry will attempt to parse name and version from this string, if they are not explicitly given.

From https://develop.sentry.dev/sdk/event-payloads/contexts/#runtime-context

What do I have to do for that?

Reply:
<img width="428" alt="Screenshot 2021-05-18 at 09 23 04" src="https://user-images.githubusercontent.com/5731772/118609053-a8e03c80-b7ba-11eb-9340-fbf839e2bea3.png">

I guess giving name and version is enough if the Platform does not offer a description, so server infers it automatically

Reply:
`Platfom.version` returns a string in the form of `2.12.3 (stable) (Wed Apr 14 11:02:39 2021 +0200) on "android_arm64"`.
 So ideally I would like that the `Version` field just contains `2.12.3 (stable)`.

Or at least that what I thought how it works after reading the documentation.

Reply:
yes, that's ideal when you can grab the version separately, but if in Flutter you just get this huge String, and you can't guarantee that the format is always the same, its just hard to parse it.
eg this String might have a different format for different platforms, that's why setting such plain String isn't that bad.